<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="TS_SAMTestSuite" time="2" tests="0" failures="0" errors="1">
   <testsuite name="TS_SAMTestSuite" tests="0" failures="0" errors="1" time="2" timestamp="2019-08-18 09:56:02" hostname="sony - Liku" id="Test Suites/TS_SAMTestSuite">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="C:\\Users\\sony\\Desktop\\Freelancer\\Katalon Project\\SAM_UIAutomation\\Reports\\TS_SAMTestSuite\\20190818_095559"/>
         <property name="logFiles" value="C:\\Users\\sony\\Desktop\\Freelancer\\Katalon Project\\SAM_UIAutomation\\Reports\\TS_SAMTestSuite\\20190818_095559\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="sony - Liku"/>
         <property name="hostAddress" value="169.254.148.186"/>
         <property name="os" value="Windows 7 64bit"/>
         <property name="katalonVersion" value="6.1.3.3"/>
      </properties>
      <testcase name="Test Cases/SAM_TestCase/TC_EventCreate" classname="Test Cases/SAM_TestCase/TC_EventCreate" status="ERROR">
         <error type="ERROR" message="Test Cases/SAM_TestCase/TC_EventCreate FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/Environments' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/Environments' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:122)
	at com.kms.katalon.core.testdata.TestDataFactory$findTestData.callStatic(Unknown Source)
	at common.CommonLibrary.navigateToURL(CommonLibrary.groovy:25)
	at common.CommonLibrary.invokeMethod(CommonLibrary.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC_EventCreate.run(TC_EventCreate:16)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:336)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:327)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:306)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:232)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1566102359133.run(TempTestSuite1566102359133.groovy:35)
"/>
         <system-out><![CDATA[2019-08-18 09:56:03 - [TEST_CASE][ERROR] - Test Cases/SAM_TestCase/TC_EventCreate: Test Cases/SAM_TestCase/TC_EventCreate FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/Environments' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/Environments' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:122)
	at com.kms.katalon.core.testdata.TestDataFactory$findTestData.callStatic(Unknown Source)
	at common.CommonLibrary.navigateToURL(CommonLibrary.groovy:25)
	at common.CommonLibrary.invokeMethod(CommonLibrary.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC_EventCreate.run(TC_EventCreate:16)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:336)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:327)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:306)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:232)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1566102359133.run(TempTestSuite1566102359133.groovy:35)


2019-08-18 09:56:04 - [TEST_STEP][ERROR] - common.CommonLibrary.navigateToURL(): Cannot find test data with id 'Data Files/Environments' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/Environments' does not exist'

2019-08-18 09:56:04 - [MESSAGE][ERROR] - Cannot find test data with id 'Data Files/Environments' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/Environments' does not exist']]></system-out>
         <system-err><![CDATA[2019-08-18 09:56:03 - [TEST_CASE][ERROR] - Test Cases/SAM_TestCase/TC_EventCreate: Test Cases/SAM_TestCase/TC_EventCreate FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/Environments' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/Environments' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:122)
	at com.kms.katalon.core.testdata.TestDataFactory$findTestData.callStatic(Unknown Source)
	at common.CommonLibrary.navigateToURL(CommonLibrary.groovy:25)
	at common.CommonLibrary.invokeMethod(CommonLibrary.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC_EventCreate.run(TC_EventCreate:16)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:336)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:327)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:306)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:232)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1566102359133.run(TempTestSuite1566102359133.groovy:35)]]></system-err>
      </testcase>
      <system-out><![CDATA[2019-08-18 09:56:02 - [TEST_SUITE][ERROR] - TS_SAMTestSuite: Test Cases/SAM_TestCase/TC_EventCreate FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/Environments' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/Environments' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:122)
	at com.kms.katalon.core.testdata.TestDataFactory$findTestData.callStatic(Unknown Source)
	at common.CommonLibrary.navigateToURL(CommonLibrary.groovy:25)
	at common.CommonLibrary.invokeMethod(CommonLibrary.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC_EventCreate.run(TC_EventCreate:16)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:336)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:327)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:306)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:232)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1566102359133.run(TempTestSuite1566102359133.groovy:35)]]></system-out>
      <system-err><![CDATA[2019-08-18 09:56:02 - [TEST_SUITE][ERROR] - TS_SAMTestSuite: Test Cases/SAM_TestCase/TC_EventCreate FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/Environments' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/Environments' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:122)
	at com.kms.katalon.core.testdata.TestDataFactory$findTestData.callStatic(Unknown Source)
	at common.CommonLibrary.navigateToURL(CommonLibrary.groovy:25)
	at common.CommonLibrary.invokeMethod(CommonLibrary.groovy)
	at com.kms.katalon.core.main.CustomKeywordDelegatingMetaClass.invokeStaticMethod(CustomKeywordDelegatingMetaClass.java:49)
	at TC_EventCreate.run(TC_EventCreate:16)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:336)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:327)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:306)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:232)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1566102359133.run(TempTestSuite1566102359133.groovy:35)]]></system-err>
   </testsuite>
</testsuites>
